pkgs/development/python-modules/refery/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+
+, poetry-core
+, pyyaml
+, colorama
+, junit-xml
+}:
+
+buildPythonPackage rec {
+  pname = "refery";
+  version = "2.1.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "daafdc841e2293db12dcf11bcc38605fbc347a774ee3af1cb23e60cd3a75f6a2";
```

```suggestion
    hash = "daafdc841e2293db12dcf11bcc38605fbc347a774ee3af1cb23e60cd3a75f6a2";
```
it's preferred to use hash over sha256 if the fetcher supports it

---

This is resolved by d04bb2d74f720f1ba1d78850e0300327be05d49f. I had to change the hash to make sure it follows SRI.